main.nf

```diff
@@ -25,6 +25,7 @@ params.production = false
 params.schema_ignore_params = ""
 params.help = false
 params.step = "update_potential_phi_fields_table"
+params.use_grs = false
```

Comment:
Will want to let Chelsea know that she will have to set `use_grs` to true for some of the cohort release we have to do in the coming months.

Reply:
yes will add to SOP once merged